recipe/activate.bat

```diff
@@ -66,16 +66,18 @@ if errorlevel 1 (
 )
 
 IF "@cross_compiler_target_platform@" == "win-64" (
+  set "target_platform=amd64"
```

Comment:
note the change [here](https://github.com/AnacondaRecipes/vs2022-feedstock/commit/6d4586dd6d7a9b1ee0e6436824407e970bd0b92d), this fixed something that I can't remember